packages/icons/icons.json

```
@@ -2606,161 +2606,161 @@
     {
         "displayName": "Path",
         "iconName": "path",
-        "tags": "hierarchy, trail, steps",
+        "tags": "hierarchy, trail, steps, sort, contour, ladder, zipper, drill down, hierarchy",
         "group": "interface",
         "codepoint": 62062
     },
     {
         "displayName": "Moon",
         "iconName": "moon",
-        "tags": "night, sky, dark",
+        "tags": "night, sky, dark, sleep, do not disturb, night time, crescent",
         "group": "miscellaneous",
         "codepoint": 62025
     },
     {
         "displayName": "Remove column",
         "iconName": "remove-column",
-        "tags": "table, detach, delete",
+        "tags": "table, detach, close sidebar, collapse, close, delete left",
         "group": "table",
         "codepoint": 62093
     },
     {
         "displayName": "Numerical",
         "iconName": "numerical",
-        "tags": "numbers, order, sort, arrange, array",
+        "tags": "numbers, order, sort, arrange, array, counting, data, integers",
         "group": "action",
         "codepoint": 62048
     },
     {
         "displayName": "Key: tab",
         "iconName": "key-tab",
-        "tags": "interface, shortcuts, buttons",
+        "tags": "interface, shortcuts, buttons, both ways, two way, push out",
         "group": "media",
         "codepoint": 61972
     },
     {
         "displayName": "Regression chart",
         "iconName": "regression-chart",
-        "tags": "graph, line, chart",
+        "tags": "graph, line, chart, scatter, plot, trend, correlation, mean, average",
         "group": "data",
         "codepoint": 62090
     },
     {
         "displayName": "Translate",
         "iconName": "translate",
-        "tags": "language, convert",
+        "tags": "language, convert, character, mandarin, japanese, translation, conversion",
         "group": "action",
         "codepoint": 62201
     },
     {
         "displayName": "Eye: on",
         "iconName": "eye-on",
-        "tags": "visibility, show",
+        "tags": "visibility, show, visible, check, verify, accept, accessibility, vision, verified, seen, viewed",
         "group": "interface",
         "codepoint": 61858
     },
     {
         "displayName": "Vertical bar chart: ascending",
         "iconName": "vertical-bar-chart-asc",
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, increasing, upward trend, signal strength, inreasing",
         "group": "data",
         "codepoint": 62220
     },
     {
         "displayName": "Horizontal bar chart: ascending",
         "iconName": "horizontal-bar-chart-asc",
-        "tags": "graph, bar, histogram",
+        "tags": "graph, histogram, sort, ranking, stats, review",
```

not sure what this has to do with "review"

```suggestion
        "tags": "graph, histogram, sort, ranking, stats",
```